src/io/iomanager.cpp

```diff
@@ -510,6 +526,66 @@ void IOManager::parse_message_cb(uint32_t msg_type, client_handler& client){
             error_message += "IOManager Parse Message: Used MsgType_INT_MAX_SENTINEL_DO_NOT_USE_ as Msg Type. ";
             break;
         }
+        case ::missiondmx::fish::ipcmessages::MSGT_REMOVE_FADER_BANK_SET:
+            try {
+                missiondmx::fish::ipcmessages::remove_fader_bank_set msg;
+                if(!msg.ParseFromZeroCopyStream(buffer)) {
+                    error_message += "Failed to decode MSGT_REMOVE_FADER_BANK_SET message.";
+                }
+                if(control_desk_handle) {
```

Comment:
I thought i had (only) looked at iomanager, but here I dont understand your success/failure logic (my my code above is not nice as well) , but here you go ahead, also when  parsing fails...
and you dont return, so you get the error message for unknown msg types (with that message length)

Reply:
Line 558 features a break instruction, hence the error handling logic is correct. But the missing return is actually a bug and soon to be fixed.

Reply:
Fixed with commit 1b70296.